test/database/db/routeSearch/postgresRouteSearchDataSource.spec.ts

```diff
@@ -58,4 +58,90 @@ describe('PostgresRouteSearchDataSource', () => {
       });
     });
   });
+
```

Comment:
Mano ficou bala demais!
Acho que só como sugestão a gente pudesse criar umas constantes, ja que não estamos mudando algumas infos. Por exemplo:
```javascript
  const SEARCH_QUERY_BASE = "SELECT * from ...."
  const SEARCH_FILTERS = {
    idCidadeOrigem: 1,
    idCidadeDestino: 2,
    idCid: 3,
    dataInicio: '2022-01-01',
    dataFim: '2022-01-02',
  };

```
E ai no `it` do test seria algo do tipo
```javascript
    const searchQuery = `${SEARCH_QUERY_BASE} WHERE s.id_cidade_origem = ${SEARCH_FILTERS.idCidadeOrigem}`;
```
Mas assim é bem opcional mesmo, foi a única sugestão que pensei pra esse código kkkkk
De qualquer forma, ta bala demais já